src/components/AnnotationPopover/AnnotationPopover.js

```diff
@@ -58,6 +58,18 @@ class AnnotationPopover extends React.PureComponent<Props> {
         position();
     }
 
+    getHotkeyConfigs() {
+        const { onCancel } = this.props;
+        return [
+            new HotkeyRecord({
+                description: 'Close popover',
+                key: 'esc',
+                handler: () => onCancel(),
```

Comment:
does this need to be an anonymous function?

Reply:
Nope! Removing it now